docs/commands.md

```diff
@@ -0,0 +1,87 @@
+---
+id: commands
+title: "Commands"
+---
+
+`Command[Model]` is a description of a CLI command. This is parsed by `CliApp` to produce a collection of valid commands and transform a valid input from a user into an instance of type `Model`. Then you can implement the functionality of the CLI App using pattern matching on instances of `Model`.
```

Comment:
In the context of a CLI app the user is familiar with, I would define what a command is and why it's useful. Walk the user through several examples of commands.

In the "home page" (index.md) we need to introduce all the main concepts, including commands, options, and arguments, but also in each section, we need to introduce more lengthy explanations and definitions, with examples, so the user has a very clear understanding.